ui/src/app/pages/workspace/workspace-wrapper/component.ts

```diff
@@ -156,9 +156,9 @@ export class WorkspaceWrapperComponent implements OnInit, OnDestroy {
         try {
           await LeoRuntimeInitializer.initialize({
             workspaceNamespace: workspace.namespace,
-            onPoll: (runtime) => {
-              runtimeStore.set({runtime: runtime, workspaceNamespace: workspace.namespace});
-            },
+            // onPoll: (runtime) => {
```

Comment:
I think the `runtimeStore` and the companion `runtime-utils` code should be added to the `leo-runtime-initializer`. This way we don't need to export `runtimeStore`. I think modifications to this store should go through API's (e.g. hooks or wrappers) otherwise things can get out of sync easily - directly accessing the store without setting up the correct workspace can cause an issue.

Reply:
I don't understand why this was here - just an oversight? LeoRuntimeInitializer already sets the store value on poll. If so, removing this seems correct